PowerUp/app/src/main/res/layout-land/completed_scene.xml

```diff
@@ -18,23 +18,23 @@
 
     <Button
         android:id="@+id/ContinueButtonMap"
-        android:layout_width="wrap_content"
+        android:layout_width="@dimen/ContinueButtonMap_width"
         android:layout_height="wrap_content"
         android:layout_alignParentBottom="true"
-        android:layout_alignParentEnd="true"
-        android:layout_alignParentRight="true"
-        android:layout_marginBottom="@dimen/completed_scene_continue_button_map_land_margin_bottom"
+        android:layout_alignLeft="@+id/completed_scene_textView"
+        android:background="@drawable/brown_button_background"
+        android:layout_marginBottom="@dimen/ContinueButtonMap_margin_bottom_landscape"
         android:layout_marginRight="@dimen/completed_scene_continue_button_map_land_margin_right"
```

Comment:
start with small letters in all the id's and tag names for resources taken from dimen